dev/bots/docs.sh

```diff
@@ -107,16 +107,25 @@ function parse_args() {
   fi
 }
 
+function build_snippets() (
```

Comment:
nit:

```suggestion
function build_snippets_tool() (
```

(to avoid confusion in the future that this is not building the actual snippets)